README.md

```diff
@@ -114,7 +114,34 @@ val options = RTVIClientOptions(
 
 val client = RTVIClient(transport, callbacks, options)
 
-client.start().withCallback {
+client.connect().withCallback {
+    // ...
+}
+```
+
+
+## Small WebRTC Transport
+
+Add the following dependency to your `build.gradle` file:
+
+```
+implementation "ai.pipecat:small-webrtc-transport:0.3.5"
+```
+
+Instantiate from your code:
+
+```kotlin
+val options = RTVIClientOptions(
+    params = RTVIClientParams(baseUrl = null),
+    enableMic = true,
+    enableCam = true
+)
+
+val connectionUrl = "http://localhost:7860/api/offer"
+
+val client = RTVIClient(SmallWebRTCTransport.Factory(context, connectionUrl), callbacks, options)
```

Comment:
Maybe in the future, we’ll need to change this so we receive the `connectionUrl` as a result from the `authBundle`, kind of the same way it’s happening with Daily right now.
But if we get to that point, we’ll also need to change the iOS SDK and Web SDK.
For now, this is the way to go.